pkg/controller/vsphere/machine_scope.go

```diff
@@ -139,21 +137,7 @@ func (s *machineScope) GetUserData() ([]byte, error) {
 	return userData, nil
 }
 
-// This is a temporary assumption to expose credentials as a secret
-// TODO: re-evaluate this when is clear how the credentials are exposed
-// for us to consume
-//
-// expects:
```

Comment:
Rather than dropping this can we update it to reflect the new assumption and possibly a link to https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/k8s-secret.html?